samtools.wdl

```diff
@@ -307,7 +315,9 @@ task View {
         File inFile
         File? referenceFasta
         String outputFileName = "view.bam"
-        Boolean? uncompressedBamOutput
+	  Boolean? includeHeader
+        Boolean? outputBam
```

Comment:
Booleans shouldn't be optional, use a default of false instead

Reply:
I don't think I wrote this. I only added the task FilterShortReadsBam. 
Should I adjust it anyways?

Reply:
yes